src/clarity-angular/buttons/_buttons.clarity.scss

```diff
@@ -215,6 +241,64 @@ $clr-button-appearance-map: (
         @include generateButton();
         @include clr-getStylesFromMap($clr-button-appearance-map, form);
         @include populateButtonProperties(default);
+    }
+
+    .btn-group {
+        //.btn-outline-<state> is the new BS4 naming convention
+        //TODO: we support both the previous one and the new one but we will deprecate the old ones from the example in 0.5.0
+        &.btn-secondary > .btn,
```

Comment:
Since we aren't getting any real efficiency out of duplicating the `btn-*` classes here, I would prefer to change it so that they follow a pattern similar to SuitCSS. So...

`.btn-group--success`, `.btn-group--primary`, etc.

That way we can remove all the `&.btn-*` selectors and go straight to `.btn-group--*` selectors.

That seems cleaner to me.